packages/jest/package.json

```
@@ -42,17 +42,17 @@
     "ci": "./scripts/ci.sh"
   },
   "dependencies": {
-    "@angular-devkit/architect": ">=0.1600.0 < 0.1700.0",
-    "@angular-devkit/core": "^16.0.0",
+    "@angular-devkit/architect": ">=0.1700.0-rc.2 < 0.1800.0",
+    "@angular-devkit/core": "^17.0.0-rc.2",
     "jest-preset-angular": "13.1.2",
```

Official support landed in 13.1.3.
```suggestion
    "jest-preset-angular": "13.1.4",
```